pkgs/applications/graphics/sane/backends/default.nix

```diff
@@ -16,17 +16,19 @@
 
 stdenv.mkDerivation {
   pname = "sane-backends";
-  version = "1.0.32";
+  version = "1.1.1";
 
   src = fetchurl {
     # raw checkouts of the repo do not work because, the configure script is
     # only functional in manually uploaded release tarballs.
     # https://gitlab.com/sane-project/backends/-/issues/440
     # unfortunately this make the url unpredictable on update, to find the link
     # go to https://gitlab.com/sane-project/backends/-/releases and choose
-    # the link with other in the URL.
-    url = "https://gitlab.com/sane-project/backends/uploads/104f09c07d35519cc8e72e604f11643f/sane-backends-1.0.32.tar.gz";
-    sha256 = "055iicihxa6b28iv5fnz13n67frdr5nrydq2c846f9x7q0vw4a1s";
+    # the link under the heading "Other".
+    src = fetchurl {
```

Comment:
Seems like this line got duplicated

Reply:
ahah I had fixed it locally but forgot to force push... thanks!